modules/ROOT/pages/local_development.adoc

```diff
@@ -0,0 +1,139 @@
+include::{partialsdir}/attributes.adoc[]
+
+= Local Development
+
+To facilitate the development of mobile services on a users 
+local machine or trialling {product-name} without requiring 
+extra hardware, it is possible to bring up {product-name} on 
+your local machine. This guide takes you through the process 
+of setting up {product-name} locally.
+
+== Requirements
+
+=== Ansible
+
+Ansible version 2.3 or later is a requirement of running the 
+installation scripts on your local machine, follow the 
+installation steps
+link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[here].
+
+=== Node.js
+
+Node.js version 8 or later and the bundled package manager 'npm' 
+are a requirement for certain scripts in the installation process, 
+install these with the instructions:
+link:https://nodejs.org/en/download/package-manager/[here].
+
+=== OpenShift Client Tools
+
+The OpenShift client tools are a requirement and must be 3.9 or greater,
+they can be downloaded from 
+link:https://www.openshift.org/download.html[here].
+
+=== Mobile-core Installer
+
+We configure the local development installation of {product-name}
+using ansible scripts in our link:https://github.com/aerogear/mobile-core[mobile-core] repo.
+
+Clone this repo to your local machine and check out the 1.0.0-alpha tag using:
+```
+git clone git@github.com:aerogear/mobile-core.git
+git checkout 1.0.0-alpha
+```
+
+=== Docker
+
+Docker is a requirement, and must be a specific version depending
+on your OS in order to be compatible with the latest release of
+OpenShift.
+
+==== Darwin / MacOSX
+
+On Darwin / MacOSX operating systems, you may install the latest version of 
+link:https://store.docker.com/editions/community/docker-ce-desktop-mac[Docker].
+
+==== Linux
+
+On Linux you must install version 17.09.1 of Docker which is available 
+link:https://download.docker.com/linux/centos/7/x86_64/stable/Packages/docker-ce-17.09.1.ce-1.el7.centos.x86_64.rpm[here].
+
+=== Installation
+
+The installation process is straightforward when using the `install.sh` 
+script - which is available in the mobile-core repo we have already
+cloned in the requirements section - with the following command:
+```
+./installer/install.sh
+```
+
+This validates that ansible, npm, docker and the OpenShift Client Tools
+are installed and the right version.
+
+Once verified it prompts you for your docker hub login credentials, which
+must be valid for the installation to continue.
+
+The next set of values are optional and can be left with the default values:
+```
+DockerHub Tag (Defaults to 1.0.0-alpha): 
+DockerHub Organisation (Defaults to aerogearcatalog): 
+Cluster IP (Defaults to 192.168.37.1): 
```

Comment:
Optional note: point cluster to external network ip so mobile devices could see it.